lib/mapper.js

```diff
@@ -576,7 +576,7 @@ module.exports = (function () {
 				dynamic = self._dynamic[parentPath || '.'];
 
 				// keep the value if the dynamic mapping is true (the default setting)
-				if (typeof dynamic === 'undefined' || dynamic === true) {
+				if (dynamic === true) {
 					return false;
 				}
 
```

Comment:
@brozeph Let me know if this is correct. I made the assumption that once we are at this point (where the fieldPath is not defined in the mapping), that if dynamic is `undefined`, we can assume it is not supposed to exist in the model and so we should not treat it as the equivalent of dynamic = `true`.